test/py/test_module_construct.py

```diff
@@ -1,22 +1,34 @@
-import migraphx
+import migraphx, array, sys
+try:
+    import numpy as np
+except:
+    sys.exit()
```

Comment:
I dont think its good to skip all the tests if numpy is missing. Ideally we should move all our numpy tests to a seperate file. You could start by adding this same test in a test_numpy.py file.

Reply:
Moved that numpy test to seperate file.